### DISCUSSION TITLE
Generated jsdoc has /scripts/third-party folder using out-of-date libraries and some versions are not listed

### DISCUSSION THREAD

**Discussion by jjfrankovich** [ankitskvmdam/clean-jsdoc-theme#300] [General]
Hello,
I was checking out different jsdoc themes and this one looks great.

I started to generate the jsdoc for my module and I noticed the various js files in third-party folder that is part of the generated files. Thanks for organizing those in their own folder

I am now trying to track down all the version in the /scripts/third-party folder that gets created.

It seems that some of the libraries are out-of-date and some version numbers are missing, is there someone who can clarify what the versions are, and is there someone can can clarify how often these third party libraries can be updated?
If there is no plan to update these, is there a way to tell clean-jsdoc to just use the latest version when its generating the doc?

Here are the version numbers I have tracked down so far:

    Fuse.js v6.4.6 (there is a 7.0.0 available)
    Highlight.js 11.0.0-beta1 (11.10.0 is available)
    popper.js/core v2.11.5 (2.11.8 is available)
    tippy.js version is unknown (6.2.5 is available)
    tocbot.js version is unknown (2.2.1 is available)

Thanks!

**Comment by ankitskvmdam** [comment]
v5 is a ground-up rewrite and no longer ships the old bundled `/scripts/third-party` folder, so those out-of-date libraries are gone. Closing as resolved by v5. Upgrade guide: https://ankdev.me/clean-jsdoc-theme/theme/migrate-v4-to-v5/
